### ISSUE TITLE
Missing Module?

### ISSUE THREAD

**Issue by harryhowiefish** [peterbraden/node-opencv#232]
I have OpenCV 2.4.10 on Mac OS 10.10
It seems like I successfully installed node-opencv
but when I run any one of the examples
I get the following error

```
module.js:338
   throw err;
        ^
Error: Cannot find module '/Users/HarryLin/node_modules/opencv/build/opencv/v3.0.0/Release/node-v14-darwin-x64/opencv.node'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/HarryLin/node_modules/opencv/lib/bindings.js:4:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

**Comment by cristiandley** [comment]
@harryhowiefish i have the same issue, no clue  at all... wich version of node are you using ?

**Comment by harryhowiefish** [comment]
@cristiandley 
I'm using 0.12.0

**Comment by Crymsius** [comment]
Same issue here on a mac (yosemite). Even isolating the script and the files do not work (or maybe I misunderstood the instructions ?...)

**Comment by iginia** [comment]
Having this issue with OpenCV 2.4.13, Node 0.12.15 and Mac OS 10.11.5. Anyone having the same issue ?

**Comment by imomin** [comment]
Following worked for me.
npm config set python /usr/bin/python

You may have to uninstall and reinstall opencv.

**Comment by danschultzer** [comment]
> Error: Cannot find module '/home/cristian/proyectos/test/ocv/build/opencv/v3.0.1/Release/node-v11-linux-x64/opencv.node'

This sounds like the opencv bindings was not built, or built for another node version than what you use. The former should be fixable with a reinstall of `node_modules`, and the latter has been fixed with #437. Do anyone still experience this issue?

**Comment by babenzele** [comment]
@danschultzer I just installed for the first time and this happened to me. A reinstall of node_modules fixed it. 

Note that I tried to install in node before I installed the opencv package in brew. When it failed I read the docs and realize my mistake then installed opencv with brew, then installed in node and it printed out like it worked but took no time at all, uninstalled in node, installed in node taking some time to build, working.
